.github/workflows/r_solaris.yml

```diff
@@ -19,7 +19,6 @@ jobs:
           apt-get update
           apt-get install --no-install-recommends -y \
             curl \
-            git \
```

Comment:
`git` already exists in this image.

```shell
docker run --entrypoint="" -it wch1/r-debug which git
# /usr/bin/git
```

Reply:
Then this simplification can be done for `r_valgrind` job as well (not necessary to do it in the same PR):
https://github.com/microsoft/LightGBM/blob/8deb90948a481820ba2109166038b146ae63a5dd/.github/workflows/r_valgrind.yml#L12-L23

Reply:
oh nice! I'll push that change here.